reach4help/reach4help
# Added Google Authentication for Login

## Description
#761 Added a way for people to login with Google on top of Facebook, added styling for login buttons
## Motivation
Provide more options for users to access application now with Google + Facebook
## Testing Guidelines
Sign in with both methods
Note that you will receive this error if you try logging on using both auth methods with the same email: 
![image](https://user-images.githubusercontent.com/26425671/84846018-bd441400-b002-11ea-84de-4d62899dc459.png)

## Release Checklist

- [y] I have a plan to verify that these changes are working as expected after deploy
- [y] I have a plan for how to revert, rollback, or disable these changes if things are not working as expected

## Security Checklist

This PR creates, modifies, or deletes:

- [y] Authentication: Authentication mechanism



## Additional Notes
closes #761 
![image](https://user-images.githubusercontent.com/26425671/84846140-fbd9ce80-b002-11ea-8c9c-fd6e8b7d733d.png)
